integreat_cms/cms/views/dashboard/dashboard_view.py

```diff
@@ -169,13 +169,18 @@ def get_low_hix_value_context(self) -> dict[str, QuerySet]:
         if not settings.TEXTLAB_API_ENABLED or not self.request.region.hix_enabled:
             return {}
 
-        translations_under_hix_threshold = PageTranslation.objects.filter(
+        translations = PageTranslation.objects.filter(
             language__slug__in=settings.TEXTLAB_API_LANGUAGES,
             id__in=self.latest_version_ids,
             page__hix_ignore=False,
```

Comment:
```suggestion
            page__hix_ignore=False,
            hix_score__lt=settings.HIX_REQUIRED_FOR_MT
```
It would probably make sense to speed this up a bit, since we have a lot of page translations on the production server and probably want to avoid iterating through many